ixmp4/core/iamc/variable.py

```diff
@@ -75,9 +72,8 @@ def tabulate(self, name: str | None = None) -> pd.DataFrame:
         return self.backend.iamc.variables.tabulate(name=name)
 
     def _get_variable_id(self, variable: str) -> int | None:
-        if variable is None:
-            return None
-        elif isinstance(variable, str):
+        # TODO this check seems kind of redundant...
+        if isinstance(variable, str):
```

Comment:
Hm, so I think we are generally missing many argument checks for users without mypy. Since this is an internal function I would remove the check here completely and add checks in the user-facing core layer functions at some point(tm)